tesseract.yaml

```diff
@@ -0,0 +1,39 @@
+name: tesseract
+config-opts:
+  - --disable-static
+  - --disable-graphics
+  - --enable-legacy
+sources:
+  - type: archive
+    url: https://github.com/tesseract-ocr/tesseract/archive/4.1.1.tar.gz
+    sha256: 2a66ff0d8595bff8f04032165e6c936389b1e5727c3ce5a27b3e059d218db1cb
+  # The LSTM models (--oem 1) in these files have been updated to the integerized versions of tessdata_best on GitHub.
+  # So, they should be faster but probably a little less accurate than tessdata_best.
+  #- type: archive
+  #  url: https://github.com/tesseract-ocr/tessdata/archive/4.0.0.tar.gz
+  #  sha256: 38c637d3a1763f6c3d32e8f1d979f045668676ec5feb8ee1869ee77cedd31b08
+  #  dest: traineddata
+  # Fast integer versions of trained models
+  #- type: archive
+  #  url: https://github.com/tesseract-ocr/tessdata_fast/archive/4.0.0.tar.gz
+  #  sha256: f1b71e97f27bafffb6a730ee66fd9dc021afc38f318fdc80a464a84a519227fe
+  #  dest: traineddata
+  # Best (most accurate) trained LSTM models.
+  #- type: archive
+  #  url: https://github.com/tesseract-ocr/tessdata_best/archive/4.0.0.tar.gz
+  #  sha256: 5d826c8a926d06b4e19eae391949f24f3011973cc7f679c57a059054fa2d8c60
+  #  dest: traineddata
+  - type: git
+    url: https://github.com/tesseract-ocr/tessdata_best.git
+    #tag: 4.0.0
+    commit: 9e8aeef07ce8c4f6e6519577cee76363246bc6de
+    dest: traineddata
+  # Tweak location of tessdata folder
+  # https://src.fedoraproject.org/rpms/tesseract/blob/7ae0e18c8f30353dea96807a4a445d1b3aba8d74/f/tesseract_datadir.patch
+  - type: patch
+    path: tesseract-4.1.1-datadir.patch
+post-install:
+  - install -p -D -m 0644 "traineddata"/*.traineddata -t "${FLATPAK_DEST}/share/tesseract/tessdata/";
+  - install -p -D -m 0644 "traineddata/script"/*.traineddata -t "${FLATPAK_DEST}/share/tesseract/tessdata/script/";
+modules:
+  - leptonica.yaml
```

Comment:
this is missing. Note that having separate manifest is not a good practice. This applies to the place you copied this from.